framework/resources/Functional/table_stats/stats/positive/stats_regex.json

```diff
@@ -0,0 +1,31 @@
+{
+    "testId": "Table_Stats_Analyze",
+    "type": "group",
+    "description": "Test Table Stats",
+    "categories": [
+        "smoke"
+    ],
+    "matrices": [
+        {
+            "query-file": ".*.q",
```

Comment:
Can we please move all tests to use .sql? Here i think there is a mix of both. You can use sub-dirs if you need